### PR TITLE
Javascript/javascript3/week2

### PR DESCRIPTION
**Pull request by soujanyapolavarapu**

### CODE REVIEW

File: javascript/javascript3/week2/Rewrite-time.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+getCurrentLocation()
+  .then((position) => {
+    const latitude  = position.coords.latitude;
+    const longitude = position.coords.longitude;
+    console.log(`Latitude: ${latitude} °, Longitude: ${longitude} °`);
+  })
+  .catch((error) => {
+    // called if there was an error getting the users location
+    console.log('error');
+  });
```
Comment (ccshah298) [inline javascript/javascript3/week2/Rewrite-time.js:40]: can u pl remove commented code...:)

File: javascript/javascript3/week2/main.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+translateOneByOne();
+
+function translateAllAtOnce(){
+  Promise.all([moveElement(document.querySelector("ul.marks li:nth-child(1)"), { x: 20, y: 300 }),
+               moveElement(document.querySelector("ul.marks li:nth-child(2)"), { x: 400, y: 300 }),
+               moveElement(document.querySelector("ul.marks li:nth-child(3)"), { x: 400, y: 20 })]).then((values) => {
+   console.log('All elements has been moved');
+ });
+}
+translateAllAtOnce();
```
Comment (ccshah298) [inline javascript/javascript3/week2/main.js:25]: good job 👍 ...

File: javascript/javascript3/week2/movies.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+//Creat an array of bad movies since year 2000
+fetch(moviesUrl)
+.then(response=>response.json())
+.then(movies=>{
+  console.log(movies);
+const badMovies =  movies.filter(movie=>movie.rating<3);
+const badMoviesFrom2000 = badMovies.filter(movie=>movie.year>=2000);
+    console.log(badMovies);
+    console.log(badMoviesFrom2000);
+});
```
Comment (ccshah298) [inline javascript/javascript3/week2/movies.js:15]: are u using prettier format for your vs code?  its very handy n helpful extension ...

File: javascript/javascript3/week2/main.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+// This code will move the li to the position 100, 100. Calling moveElement will return a promise that resolves after the li element has been moved.
+
+function translateOneByOne(){
+    moveElement(document.querySelector("ul.marks li:nth-child(1)"), { x: 20, y: 300 }).then(() => {
```
Comment (zuhairtaha) [inline javascript/javascript3/week2/main.js:4]: You can use `redBox, blueBox, greenBox` variables that already declared in `move-element.js` file

File: javascript/javascript3/week2/Rewrite-time.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+.catch((err)=>{
+  console.log('error');
+});*/
+
+
+function getCurrentLocation(){
+  const geolocationPromise = new Promise((resolve,reject)=>{
+  navigator.geolocation.getCurrentPosition((position)=>resolve(position), (error)=>reject(error));
+});
+return geolocationPromise();
```
Comment (zuhairtaha) [inline javascript/javascript3/week2/Rewrite-time.js:27]: `geolocationPromise` is not a function to call it!

**Review by zuhairtaha — CHANGES_REQUESTED** [review]

File: javascript/javascript3/week2/movies.js (hunk 3 above)
Comment (soujanyapolavarapu) [reply javascript/javascript3/week2/movies.js:15]: sure I will THankyou for your feedback :)

File: javascript/javascript3/week2/main.js (hunk 4 above)
Comment (soujanyapolavarapu) [reply javascript/javascript3/week2/main.js:4]: sure I  will,THankyou for your feedback :)

File: javascript/javascript3/week2/Rewrite-time.js (hunk 5 above)
Comment (soujanyapolavarapu) [reply javascript/javascript3/week2/Rewrite-time.js:27]: ohh sorry yes I will rectify

**Pull request merged**